packages/components/forma-36-react-datepicker/yarn.lock

```diff
@@ -734,20 +734,6 @@
     exec-sh "^0.3.2"
     minimist "^1.2.0"
 
-"@contentful/forma-36-react-components@^3.25.1":
```

Comment:
I'm not sure if this should be removed

Reply:
Running `lerna bootstrap` from root will link the locally found packages hence the difference between the `lock` files. We should commit these locks and we won't see differences when pulling and `lerna bootstrap`-ing.